monai/fl/utils/constants.py

```diff
@@ -33,11 +34,14 @@ class FlStatistics:
     #  https://github.com/Project-MONAI/MONAI/blob/df4a7d72e1d231b898f88d92cf981721c49ceaeb/monai/engines/trainer.py#L60
```

Comment:
Please use Please use a stable link instead, for example:
https://github.com/Project-MONAI/MONAI/blob/0.9.1/monai/engines/trainer.py#L60

Thanks.

Reply:
And I already moved this keys to `engines` in PR: https://github.com/Project-MONAI/MONAI/pull/4888.

Thanks.

Reply:
Thanks. Will use the new feature stats.